services/database-analyzer.js

```diff
@@ -64,7 +91,12 @@ function DatabaseAnalyzer(databaseConnection, config) {
           }
         });
 
-        return [fields, references, primaryKeys];
+        const options = {
+          underscore: isUnderscored(fields),
+          timestamps: hasTimestamps(fields),
+        };
```

Comment:
Much better like that good job. We could in a next PR detect `deletedAt` to set the paranoid options too ^^